src/Host/Client/Impl/rtvs/R/util.R

```diff
@@ -338,3 +338,30 @@ exists_on_path <- function(filename) {
     }
     return(FALSE);
 }
+
+exists_on_path_posix <- function(filename) {
+    folders <- strsplit(Sys.getenv('PATH'), ':')[[1]];
+    folders <- paste0(folders, "/", filename);
+    for (f in folders) {
+        if (file.exists(f)) {
+            return(TRUE);
+        }
+    }
+    return(FALSE);
+}
+
+exists_on_path <- function(filename) {
+    if(.Platform$OS.type == "windows"){
```

Comment:
Rather than doing the check inside the function, better do it outside, i.e.:
```r
exists_on_path <- if (.Platform$OS.type == "windows") {
  function(filename) {
    ...
``